test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -241,6 +241,10 @@ def _test_code_common(
 
 
 class TestPatternMatcher(TestPatternMatcherBase):
+    # The pattern match for this is kind of broken. It'll match if negative_slope
+    # is either a float or a tensor. I'll cc the person who wrote this test/match
+    # on the diff to see if they can help me fix it.
+    @torch._dynamo.config.patch(specialize_float=True)
```

Comment:
@yanbing-j @zhuhaozhe any chance I can get your help taking a look on this? Basically if you check out this PR and remove this line, you'll see the pattern match breaking.